### DISCUSSION TITLE
[Feature] multiple ip for immich host

### DISCUSSION THREAD

**Discussion by hargrove4570** [immich-app/immich#11308] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Symfonium has a similar feature.

The idea is that immich will attempt to access the server at one ip/fqdn first and when that fails it will try a second IP/fqdn. The use case is on a self hosted server that does not have hairpin NAT or some other complicated routing.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Duplicate of #308 / #2335

We recommend using a local DNS server.
